src/main/resources/db/changelog/changes/12-create-cartitem-table.yml

```
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: createCartItemsTable
+      author: Vasyliev
+      changes:
+        - createTable:
+            tableName: cartitems
```

```suggestion
            tableName: cart_items
```
we usually separate table names with underscore